sycl/test/warnings/warnings_deprecated.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clangxx -fsycl -sycl-std=2020 -fsycl-device-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -fsyntax-only -Wall -Wdeprecated -Wextra
+
+#include <sycl/sycl.hpp>
+
+
+using namespace sycl;
+queue q;
+
+
+class FunctorDeprecated {
+public:
+  // expected-warning@+1{{attribute 'intel::reqd_sub_group_size' is deprecated}}
```

Comment:
Should be a `clang/` test, `sycl` tests are way too expensive for that

Reply:
Moved to clang tests